### ISSUE TITLE
Upgrade to Lpeg 1.0

### ISSUE THREAD

**Issue by daurnimator** [pygy/LuLPeg#8]
Currently Lpeg 0.12 is emulated.
1.0 has been released (mostly bug fixes)

**Comment by pygy** [comment]
Yup, I've seen your call about it on the ML/in my mailbox too :-)

Actually, LuLPeg already supports arbitrary Lua values as group names. Not doing so would have required more code, and I didn't know it wasn't supposed to do it.

I'll look into the latest release for the misc changes.
